### PR TITLE
Risk based auth simple

### PR DESCRIPTION
**Pull request by lluft**
Risk based auth without the whole state saving.

### CODE REVIEW

File: app/services/castle_risk_guard.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CastleRiskGuard < Clearance::SignInGuard
+  def call
+    return next_guard unless @session.castle_authentication
```
Comment (lluft) [inline app/services/castle_risk_guard.rb]: couldn't come up with a more clever way of skipping a safeguard. It's needed unless we can guarantee that there will be an updated, lower risk score when creating a new authentication coming from the challenge or lock link

**Comment by lluft** [comment]
@brissmyr removed the SafeGuard

**Comment by brissmyr** [comment]
Looks good. 👍

**Comment by wallin** [comment]
@lluft are we gonna pick this up? or close?

**Comment by lluft** [comment]
@wallin I have no plans on picking this up in the near future.

**Pull request closed without merging**